docs/ref/contrib/gis/geos.txt

```diff
@@ -554,6 +554,11 @@ is returned instead.
     Requires GDAL. Raises :class:`~django.contrib.gis.geos.GEOSException` if
     GDAL is not available or if the SRID is ``None`` or less than 0.
 
+    .. versionchanged:: 1.9
```

Comment:
The annotation should be "self-contained" as described in [Documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features). Also, we are targeting 1.10 for new features and non-release blocking bug fixes (as far as I can tell this isn't one).